packages/sdk/app-framework/src/plugins/common/file.ts

```diff
@@ -0,0 +1,28 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { type Plugin } from '../PluginHost';
+
+export const defaultFileTypes = {
+  images: ['png', 'jpg', 'jpeg'],
+  media: ['mp3', 'mp4', 'mov', 'avi'],
+  text: ['pdf', 'txt', 'md'],
+};
+
+export type FileInfo = {
```

Comment:
this seems rather ipfs specific rather than a generic file info type? not that that is necessarily a bad thing, but makes me question it being in `common`. Perhaps warrants a TODO alluding to how this will attempt to provide a "common" files interface